composer.json

```diff
@@ -132,7 +132,7 @@
         "drupal/simple_sitemap": "^3.3",
         "drupal/simple_gmap": "^1.4",
         "drupal/shariff": "^1.5",
-        "drupal/slick": "^2.0-rc3",
+        "drupal/slick": "^2.1",
```

Comment:
Needs to be increased because we need the integration with the new `library.libraries_directory_file_finder`